src/Responses/Meta/MetaInformation.php

```diff
@@ -28,7 +28,7 @@ private function __construct(
      */
     public static function from(array $headers): self
     {
-        $requestId = $headers['x-request-id'][0];
+        $requestId = $headers['x-request-id'][0] ?? 'n/a';
```

Comment:
```suggestion
        $requestId = $headers['x-request-id'][0] ?? null;
```

Instead of `n/a` we should make this field nullable to be consistent with the rest of the package.

And please a test.